src/components/OcSearchBar.vue

```diff
@@ -8,21 +8,15 @@
     <div class="uk-width-expand uk-position-relative">
       <span v-if="icon" class="uk-form-icon">
         <oc-icon v-show="!loading" :name="icon" />
-        <div
-          v-show="loading"
-          :class="$_ocSpinner_class"
-          :aria-label="ariaLabel"
-          tabindex="-1"
-          role="img"
-        />
+        <oc-spinner v-show="loading" :size="spinnerSize" aria-label="Loading results" />
```

Comment:
I know that the aria-label was entirely broken before. Could you add a required property `loadingAccessibleLabel` (or similar naming) for this? Needs to be provided by the code that uses the searchbar.